src/commands/server/update.ts

```diff
@@ -182,65 +173,173 @@ Consider removing '--che-operator-image' to update Eclipse Che operator to the s
       || cheCluster.spec.auth.identityProviderImage) {
       let imagesListMsg = ''
 
-      const crPatch = ctx[ChectlContext.CR_PATCH] || {}
+      const resetImagesCrPatch: { [key: string]: any } = {}
       if (cheCluster.spec.server.pluginRegistryImage
-        && (!crPatch.spec || !crPatch.spec.server || !crPatch.spec.server.pluginRegistryImage)) {
+        && (!resetImagesCrPatch.spec || !resetImagesCrPatch.spec.server || !resetImagesCrPatch.spec.server.pluginRegistryImage)) {
```

Comment:
Why do you need such a lot of field checks on the Empty object: resetImagesCrPatch ?
You wrote above:
const resetImagesCrPatch: { [key: string]: any } = {}

Reply:
Thank you, these are leftover after some refactoring. I will fix.